Sources_v3/StreamChatUI/ChatChannel/ChatMessageReactionsView.swift

```diff
@@ -99,7 +99,7 @@ class MessageReactionView: UIView {
 
 // MARK: - Reactions
 
-class ChatMessageReactionsView: UIView {
+open class ChatMessageReactionsView: UIView {
```

Comment:
This should inherit from `View`, then you won't need `commonInit()` and you can use the lifecycle methods

Reply:
Left it as it is because we have a separate user-story for message reactions